src/main/features/response/views/reject-all-of-claim.njk

```diff
@@ -0,0 +1,46 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, radioGroup, saveAndContinueButton, errorSummary %}
+{% from "table.njk" import tableStart, tableEnd, row, rowWithMultipleValue %}
+
+{% set title = t('Money claim') %}
+
+{% set heading = t('I reject all of the claim') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      <form method="post" class="analytics-submit-event-trigger" data-event-action="Response: Select full admission option" data-event-label-from="option" novalidate>
+        {{ csrfProtection(csrf) }}
+
+        <legend class="visually-hidden">{{ t('I reject all of the claim') }}</legend>
+
+        {{
+          radioGroup(
+          label = 'Why do you reject all the claim?',
+          name = 'option',
+          options = [
+            {
+              label: 'I’ve already paid the claim in full',
+              value: 'alreadyPaid'
```

Comment:
Seeing this (not only here but on all such pages as well) I have a feeling we should export all constants classes as nunjucks locals and use them instead of using magic values.

Reply:
Agree, asked @rattur for the same but it got lower priority.